app/views/posts/index.html.erb

```diff
@@ -0,0 +1,75 @@
+<div class="container">
+<div class="row">
+<p id="notice"><%= notice %></p>
+<div class="center jumbotron main-banner">
+<div class="user-photo">
+<img class="user-gravatar" alt="User photo" src="/assets/underwater" />
+<h2><%= current_user.username %></h2> 
+</div>
+<div class="under-buner-menu">
+<a href="#">Lorem</a>
+<a href="#">Ipsum</a>
+<a href="#">Set</a>
+<a href="#">Loram</a>
+</div>
+</div>
+
+<div class="container">
+<div class="row">
+<p id="notice"><%= notice %></p>
+
+<div class="col-md-4">
+<% @posts.each do |post| %>
+<p>Username:<%= post.user.username %></p> 
+<p>Email:<%= post.user.email %></p> 
+<% end %>
+<button class='btn btn-primary'><%= link_to 'New Post', new_post_path %></button>
```

Comment:
You can't put a `<a />` link into a button. 
I had to change it like this 
```ruby
<%= link_to 'New Post', new_post_path, class: 'btn btn-primary' %>
```
to make it work